pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-overview-of-modules.adoc

```diff
@@ -0,0 +1,12 @@
+[id="overview-of-modules_{context}"]
+
+= Overview of modules
+[role="_abstract"]
```

Comment:
Hey @RichardHoch - I'd add a space in between the title and the role definition line. Shouldn't cause any issues the way it is, but just in case. The same applies to instances in the modules.

Reply:
Done.